src/main.py

```diff
@@ -671,6 +671,12 @@ def content(self, o):
             if o.post["nopic"] == "404": self.notfound()
             self.redirect("image?db=%s&mode=nopic" % o.dbo.name())
 
+class video_thumbnail(ASMEndpoint):
+    url = "video_thumbnail"
```

Comment:
We want this endpoint to be cacheable by Cloudflare, but they won't cache anything that has a session cookie in the response. Set ```session_cookie = False``` here